deno.json

```diff
@@ -32,5 +32,13 @@
         "wasi/testdata"
       ]
     }
+  },
+  "tasks": {
+    "test": "deno test --doc --unstable --allow-all --coverage=./cov --ignore=node/",
+    "test:faketime": "faketime '2021-12-31' deno test --unstable --allow-all datetime/test_2021_12_31.ts",
+    "test:browser": "git grep --name-only \"// This module is browser compatible.\" | grep -v \".github/workflows\" | xargs deno cache --config browser-compat.tsconfig.json",
```

Comment:
`xargs` doesn't work on Windows. Maybe we should add it to `deno task`?

Reply:
Actually, it would probably be better if we moved this to a deno script.

Reply:
If you feel like it, then sure :)

Reply:
I don't think anyone runs this command locally, but I might be wrong